app-frontend/src/app/pages/projects/edit/aoi-approve/aoi-approve.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-project">
+  <div class="sidebar-header">
+    <a class="btn sidebar-header-nav-btn" ui-sref="projects.edit">
+      <i class="icon-arrow-left"></i>
+    </a>
+    <h5 class="sidebar-title">AOI Scene Approval</h5>
```

Comment:
During project creation on the UI, should the language `Monitoring Project` be changed as well?